### DISCUSSION TITLE
which column can be referred as a sub category role?

### DISCUSSION THREAD

**Discussion by zhanghengtian** [firefly-iii/firefly-iii#5740] [Firefly III] [answered]
I have noticed that firefly iii only have one column referred to category, what if I need a sub category that under the category column. 
what should I do to achieve that?

thanks guys.

**Comment by JC5** [comment, marked as answer]
There is no subcategory in Firefly III.

**Comment by 42demian42** [comment]
I am late to the party here, but I have the same question.  And yes, I know there are no subcategories in Firefly III.  But - for me, the expense accounts are no value.  I have 10 pages of accounts that I don't need at all.  NOW - what if I used the destination account as a sub-category ???

Example - Utilities is a category for me.  I don't have Utilities-Gas, Utilities-Electric, Utilities-Internet, Utilities-Cell like I did in quicken.  But, what if I created a few accounts, "Gas, Electric, Internet, Cell" and then added assignment of this account to my rule.  I could get a report that shows my utilities and their makeshift subcategory by account
